tests/testthat/test-check_tbl_col_types.R

```diff
@@ -20,15 +20,19 @@ test_that("check_tbl_col_types works", {
   expect_snapshot(
     check_tbl_col_types(tbl, file_path, hub_path)
   )
-
-  # Check "06" location value read and validated correctly
-  hub_path <- test_path("testdata/hub")
-  file_path <- "hub-baseline/2023-04-24-hub-baseline.csv"
-  tbl <- read_model_out_file(
-    file_path,
-    hub_path
-  )
-  expect_snapshot(
-    check_tbl_col_types(tbl, file_path, hub_path)
-  )
 })
+
+test_that(
```

Comment:
As I understand, the behavior for CSV output as changed, should we add it in the documentation/NEWS?

Reply:
Sorry I'm not sure I follow. We've added the options to configure the value of this argument through the config but if that's not set (so all hubs up to now) it falls back to the previously default behaviour of auto-detection. So nothing really has changed unless you upgrade your config version and use the `output_type_id_datatype` property. Also not sure what you mean re: CSVs specifically. There hasn't been any format specific changes.